docs/src/components/home/sections/HeroSection.tsx

```diff
@@ -83,7 +83,7 @@ export const HeroSection = () => {
                 title="Cloud connected"
                 href="#authentication"
                 icon={<RiCloudWindyLine />}
-                desc="We handle the hard stuff so you can focus on UI"
+                desc="We handle the 'boring' stuff so you can enjoy the fun of UI development"
```

Comment:
Nit: Maybe just a teeeeeny tiny less word-ey? 

```suggestion
                desc="We handle the boring stuff so you can focus on UI development"
```

Reply:
I also tried:
```suggestion
                desc="We handle the boring stuff so you can focus on UI" 
```
but it makes the short word `UI` an orphan on desktop which looks a little not great :(